src/tests/ftest/util/ServerUtils.py

```diff
@@ -158,24 +158,24 @@ def stopServer(setname=None, hosts=None):
     has spawned.  Doesn't always work though.
     """
 
-    global sessions
+    global SESSIONS
```

Comment:
(pylint-global-variable-not-assigned) Using global for 'SESSIONS' but no assignment is done